dev-workflow-ui/webContent/view/home.xhtml

```diff
@@ -100,7 +89,7 @@
                   </p:link>
                 </p:column>
               </p:dataTable>
-              <h:panelGrid rendered="#{homepageIvyDevWfBean.tasksSize eq 0}">
```

Comment:
Looks like you can also delete some stuff in the `homepageIvyDevWfBean`?

Reply:
> Looks like you can also delete some stuff in the `homepageIvyDevWfBean`?

Yes. I think it's better to move the PersonalTasksModel into HomepageBean tho and completely delete the PersonalTasksBean

Reply:
Yes that sounds like a good plan :+1: